docs/Developer-guide/Building-GlusterFS.md

```diff
@@ -137,6 +141,13 @@ prefix, you might need to use 'sudo' or 'su -c' to install.
 
 		$ sudo make install
 
+NOTE: glusterfs can be installed on any target path, other than for
```

Comment:
Thanks for adding this :+1: 

A slightly better rewording:

glusterfs can be installed on any target path. However, the `mount.glusterfs` script has to be in `/sbin/mount.glusterfs` for mounting via command `mount -t glusterfs` to work. See `-t` section in `man 8 mount` for more details.

Reply:
Done